docker/Dockerfile.tpl

```diff
@@ -0,0 +1,30 @@
+# vim: syntax=dockerfile:
+# TODO use :latest when it will be available on docker hub
+FROM cahirwpz/mimiker-circleci:1.6
+
+RUN addgroup --gid ${MIMIKER_GID} mimiker && \
+    adduser --uid ${MIMIKER_UID} --disabled-password \
+    --gecos "" --force-badname \
+    --ingroup mimiker --ingroup sudo mimiker
+
+WORKDIR /home/mimiker
+
+# XXX: Ugly workaround for apt-get update -q crashing with the following error:
+# `Could not open file /var/lib/apt/lists/deb.debian.org_debian_dists_stretch-backports_main_binary-amd64_Packages.diff_Index`
+# running the command twice resolves the issue
```

Comment:
Shouldn't we add the source of this solution for the sake of some context?

Reply:
@mzr up